lib/cdk_node.star

```diff
@@ -0,0 +1,52 @@
+data_availability_package = import_module("./data_availability.star")
+
+NODE_COMPONENTS = struct(
+    sequence_sender="sequence-sender",
+    aggregator="aggregator",
+)
+
+def create_cdk_node_service_config(
+    args,
+    config_artifact,
+    genesis_artifact,
+    keystore_artifact,
+):
+    cdk_node_name = "cdk-node" + args["deployment_suffix"]
+
+    cmd = [
+        "sleep 20 && cdk-node run " +
```

Comment:
doc: can you document the reason for including `sleep 20` here, so that we don't forget this in a few months :)

Reply:
Terrible, we should come up with a better way, it's basically double the time erigon is waiting.